core/services/ocr2/plugins/functions/integration_tests/v1/internal/testutils.go

```diff
@@ -237,14 +239,16 @@ func StartNewChainWithContracts(t *testing.T, nClients int) (*bind.TransactOpts,
 		{Type: uint256Type}, // fulfillmentGasPriceOverEstimationBP
 		{Type: int256Type},  // fallbackNativePerUnitLink
 	}
+
 	var maxCallbackGasLimit = uint32(450_000)
 	var feedStalenessSeconds = uint32(86_400)
 	var gasOverheadBeforeCallback = uint32(325_000)
 	var gasOverheadAfterCallback = uint32(50_000)
 	var requestTimeoutSeconds = uint32(300)
 	var donFee = big.NewInt(0)
 	var maxSupportedRequestDataVersion = uint16(1)
-	var fulfillmentGasPriceOverEstimationBP = big.NewInt(6_600)
+	// TODO understand the exact gas price behavior on a simulated chain under load and adjust this number down
+	var fulfillmentGasPriceOverEstimationBP = big.NewInt(20_000)
```

Comment:
This seems very high. Is the exact gas price not working as we expect?

Reply:
yeah, I don't know why - need to understand the details of the simulated chain but I want to see if it passes in CI at all with this change.